### ISSUE TITLE
Question about render quality and multi-scale

### ISSUE THREAD

**Issue by Fangkang515** [ashawkey/torch-ngp#17]
<img width="699" alt="image" src="https://user-images.githubusercontent.com/34268707/155508491-9c3bc3e0-a6d3-44c2-b55d-84f3a1b3dc5b.png">

Thanks for your great work.
I noticed that the torch-ngp did not implement multi-scale, and I guess this may be the reason for the image blur. So I would like to ask if you have any plans to achieve multi-scale?
Thanks.

**Comment by ashawkey** [comment]
The real reason is instant-ngp use all train/val/test set to train, which is set to default in GUI mode now. You can try it now!

**Comment by JaminFong** [comment]
@ashawkey Thanks for your awesome work! I have tried official instant-ngp with only training data, still achieving high performance, >34 PSNR with 1 min on LEGO. I wonder if any clue has been figured out for the quality issue.

**Comment by ashawkey** [comment]
@JaminFong You are right, the performance in PSNR is still worse compared to instant-ngp (as I mentioned in readme's update logs). Honestly, I haven't figured out why for now, but I'm still trying.
If you have any idea, welcome to report in issue/pull request.

**Comment by JaminFong** [comment]
@ashawkey Thanks for the reply! I really admire your implementation work. I am also delving into ngp and happy for further discussion if any idea got.
